src/servers/trigger.ts

```diff
@@ -479,26 +480,40 @@ const listenerBria = async () => {
   baseLogger.info("bria listener started")
 }
 
+import { startServer } from "@services/ibex"
+const ibexListener = async () => {
+  startServer()
+
+  // const callback = (event) => baseLogger.info(event, "Ibex event received")
+  // const ibexEventHandler = wrapAsyncToRunInSpan({
+  //   namespace: "servers.trigger",
+  //   fn: callback // (event: IbexEvent) => baseLogger.info(event, "Ibex event received"),
+  // })
+}
+
 const main = () => {
-  listenerBria()
+  logger.info("Setting subscribers:")
```

Comment:
remove

Reply:
does this mean if we decide in the future to use the internal wallet, we will not be able to use Bria?